tests/scalers/influxdb.test.ts

```diff
@@ -57,7 +57,7 @@ test.before((t) => {
 
     // Wait for influxdb instance to be ready
     let influxdbStatus = 'false'
-    for (let i = 0; i < 15; i++) {
+    for (let i = 0; i < 25; i++) {
```

Comment:
for me, influxdb container was consistently taking ~40-45 seconds to start. This doesn't address the other issue with this test mentioned in the PR body above.